src/Algebra/Definitions/RawMonoid.agda

```diff
@@ -55,7 +55,9 @@ infixl 8 _×′_
 _×′_ : ℕ → Carrier → Carrier
 0     ×′ x = 0#
 1     ×′ x = x
-suc n ×′ x = x + n ×′ x
+suc n ×′ x = n ×′ x + x
+
```

Comment:
Should this breaking change be mentioned in the CHANGELOG?

Reply:
No need as this definition is new to this version of the library. The old version of the operator still exists in `Algebra.Operations.Monoid`.